src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java

```diff
@@ -39,39 +39,32 @@
  * <ul>
  * <li>
  * Property {@code allowClassName} - Controls whether to allow a method name to have the same name
- * as the residing class name. This is not to be confused with a constructor. An easy mistake is
- * to place a return type on a constructor declaration which turns it into a method. For example:
- * <pre>
- * class MyClass {
- *     public void MyClass() {} //this is a method
- *     public MyClass() {} //this is a constructor
- * }
- * </pre>
+ * as the residing class name. This is not to be confused with a constructor.
```

Comment:
```java
 * Property {@code allowClassName} - Controls whether to allow a method name to have the same name
 * as the enclosing class name. Setting this property {@code false} helps to avoid confusion between
 * constructors and methods.


```

Reply:
done